README.md

```diff
@@ -127,6 +127,8 @@ require("rip-substitute").setup {
 		},
 	},
 	regexOptions = {
+		-- Start with fixed strings on by default
+		literalByDefault = false,
```

Comment:
More verbose would be "startWithFixedStringsOn", renounces the need for the comment